modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -298,260 +298,299 @@ function add_dependency_permutations(o, deps) {
 		o.depends(res[i]);
 }
 
+// Define a class CBIWifiFrequencyValue that extends form.Value
 var CBIWifiFrequencyValue = form.Value.extend({
-	callFrequencyList: rpc.declare({
```

Comment:
This diff looks like your IDE/editor is formatting the tab indentations to 4 spaces, please fix it.

Reply:
(that it should be tabs, and not spaces)

Reply:
@rmandrad note: this is still an issue

Reply:
updated ... I don't understand what happened here as I have haven't any new code only additional data to cover the 11be ... anyhow pls check  if it conforms to the standards